src/Refitter.Core/SchemaCleaner.cs

```diff
@@ -0,0 +1,172 @@
+﻿using NJsonSchema;
+
+using NSwag;
+
+namespace Refitter.Core;
+
+public class SchemaCleaner
+{
+    private readonly OpenApiDocument _doc;
+
+    public SchemaCleaner(OpenApiDocument doc)
+    {
+        _doc = doc;
+    }
+    public void RemoveUnreferencedSchema()
```

Comment:
@kirides can you please add an empty line between the constructor and the `RemoveUnreferencedSchema()` method declaration

Reply:
done